### PR TITLE
Fix cache.redis usage

### PR DESCRIPTION
**Pull request by adriendupuis**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | N/A
| Versions      | 4.0, 3.3

* Fix path to `config/packages/cache_pool/`
* Update `CACHE_POOL` and `.env` usage after `config/env/` removal between 2.x and 3.x

### CODE REVIEW

File: docs/guide/persistence_cache.md [hunk 1]
```diff
@@ -59,9 +59,9 @@ The underlying cache system is exposed as an `ibexa.cache_pool` service, and can
 ### Configuration
 
 By default, configuration uses the `cache.tagaware.filesystem` service to store cache files.
-The service is defined in `bin/config/cache_pool/cache.tagaware.filesystem.yml`
+The service is defined in `config/packages/cache_pool/cache.tagaware.filesystem.yaml`
 to use [FilesystemTagAwareAdapter](https://github.com/ibexa/recipes/blob/master/ibexa/oss/4.0.x-dev/config/packages/cache_pool/cache.tagaware.filesystem.yaml#L8).
-This service is loaded through `bin/config/env/generic.php`.
+This service is loaded through `config/env/generic.php`.
```
Comment (adriendupuis) [inline docs/guide/persistence_cache.md:64]: */env/generic.php & */env/platformsh.php doesn't seem to exist since 3.3

Comment (adriendupuis) [reply docs/guide/persistence_cache.md:64]: [.env](https://symfony.com/doc/5.4/configuration.html#config-dot-env) and [vendor/platformsh/symfonyflex-bridge/platformsh-flex-env.php](https://github.com/platformsh/symfonyflex-bridge/blob/master/platformsh-flex-env.php) are now doing this part on both 3.3 and 4.x

File: docs/guide/persistence_cache.md [hunk 2]
```diff
@@ -159,26 +159,26 @@ Out of the box in `config/packages/cache_pool/cache.redis.yaml` you'll find a de
...
+    For Ibexa Cloud/Platform.sh: This is automatically configured in `vendor/platformsh/symfonyflex-bridge/platformsh-flex-env.php` if you have enabled Redis as `rediscache` Platform.sh service.
 
-For anything else, you can enable it with environment variables detected automatically by `bin/config/env/generic.php`.
+For anything else, you can enable it with environment variables.
 For instance, if you set the following environment variables `export CACHE_POOL="cache.redis" CACHE_DSN="secret@example.com:1234/13"`, it will result in config like this:
 
 ``` yaml
 services:
     cache.redis:
-        # NOTE: This optimized Redis Adapter is avaiable as of 2.5LTS via https://github.com/ezsystems/symfony-tools
```
Comment (adriendupuis) [inline docs/guide/persistence_cache.md:170]: TODO: also fix "avaiable" typo in [2.5](https://github.com/ezsystems/developer-documentation/blob/2.5/docs/guide/persistence_cache.md?plain=1#L172).

File: docs/guide/persistence_cache.md [hunk 3]
```diff
@@ -159,26 +159,26 @@ Out of the box in `config/packages/cache_pool/cache.redis.yaml` you'll find a de
...
+        # NOTE: Available via https://github.com/symfony/cache
+        class: Symfony\Component\Cache\Adapter\RedisTagAwareAdapter
         parent: cache.adapter.redis
         tags:
             - name: cache.pool
               clearer: cache.app_clearer
               provider: 'redis://secret@example.com:1234/13'
-              # Default CACHE_NAMESPACE value, see bin/config/cache_pool/cache.redis.yaml for usage with e.g. multi repo.
-              namespace: 'ez'
+              # Default CACHE_NAMESPACE value, see config/cache_pool/cache.redis.yaml for usage with e.g. multi repo.
```
[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/guide/persistence_cache.md:177]: [In OSS 3.3.16, it was `ezp`](https://github.com/ibexa/recipes/blob/master/ibexa/oss/3.3.16/manifest.json#L76)

File: docs/guide/persistence_cache.md (hunk 1 above)
Comment (adriendupuis) [reply docs/guide/persistence_cache.md]: See 6b49a79bd2e446e60d2743206329d5b9b415a558

**Review by DominikaK — APPROVED** [review]

File: docs/guide/persistence_cache.md (hunk 2 above)
Comment (adriendupuis) [reply docs/guide/persistence_cache.md:170]: Done in b8859bfe9da1c15909784e7be5a409de76d4f619

**Pull request merged**